app/scripts/views/base.js

```diff
@@ -151,8 +151,11 @@ define(function (require, exports, module) {
       NotifierMixin.initialize.call(this, options);
 
       // Prevent errors from being displayed by aborted XHR requests.
-      this._boundDisableErrors = _.bind(this.disableErrors, this);
+      this._boundDisableErrors = this.disableErrors.bind(this);
```

Comment:
Just for future reference, are we polyfilling this now then? Or (even better) have we dropped support for oldIE?